metadata-models/src/main/pegasus/com/linkedin/chart/ChartInfo.pdl

```diff
@@ -0,0 +1,52 @@
+namespace com.linkedin.chart
+
+import com.linkedin.common.AccessLevel
+import com.linkedin.common.ChangeAuditStamps
+import com.linkedin.common.DatasetUrn
+import com.linkedin.common.Time
+
+/**
+ * Information about a chart
+ */
+record ChartInfo {
+
+  /**
+   * Title of the chart
+   */
+  title: string
+
+  /**
+   * Detailed description about the chart
+   */
+  description: string
+
+  /**
+   * Captures information about who created/last modified/deleted this chart and when
+   */
+  lastModified: ChangeAuditStamps
+
+  /**
+   * Input datasets for the chart
+   */
+  inputs: optional array[DatasetUrn]
```

Comment:
🎉 very excited about this - it will be extremely powerful for us to route end-to-end lineage between dataset <-> chart <-> dashboard